src/EA.Weee.Web/Areas/Producer/Views/Producer/AlreadySubmittedAndPaid.cshtml

```diff
@@ -0,0 +1,26 @@
+﻿@using EA.Weee.Web.RazorHelpers
+@using MvcNavigationHelpers
+@model Guid
+
+@{
+    const string title = "You have already submitted your producer registration for this year";
+    ViewBag.Title = title + " - GOV.UK";
+}
+@section full_main_content
+{
+    @(this.WeeeGds().BackLink(@Url.UrlFor<EA.Weee.Web.Areas.Scheme.Controllers.HomeController>(a => a.ChooseActivity(Model))))
+
+    <header class="hgroup">
+        <h1 class="govuk-heading-l">@title</h1>
+    </header>
+
+    <p class="govuk-body">
+        You have already submitted your producer registration for the current year.
+        If you would like to view you current year submission,
```

Comment:
@seanciv  Thinking back to some screen reader issues we have had might be worth removing any newlines etc in the cshml and the text can just flow in a normal P.